reactComponents/src/searchbar/tokenSearch/SearchFilters.tsx

```diff
@@ -12,50 +10,200 @@ import {
 } from 'react-accessible-accordion';
 import { FilterNetworkAll, FilterNetworkSelectors } from "./SearchFiltersNetworkSelectors";
 import { FilterExchangeAll, FilterExchangeSelectors } from "./SearchFiltersExchangeSelectors";
-import {RootState} from "../redux/store";
+import { RootState } from "../redux/store";
+import TokenSearchContext from '../Context/TokenSearch';
+
+const FilterWrapper = styled.div`  
+  ${({styles}) => `    
+    .accordion__button {
+      position: relative;
+    }
+    background-color: ${ styles?.backgroundColor || "#00070E" };
+    border-radius: ${ styles?.borderRadius || "4px" };
+
+    .accordion__button:first-child:after {
+      display: block;    
+      content: '';
+      position: absolute;    
+      transform: rotate(-45deg);  
+      
+      color: ${ styles?.toggleColor || "#B4BBC7" };
+      height: ${ styles?.toggleHeight || "7px" };
+      width: ${ styles?.toggleWidth || "7px" };
+      margin-right: ${ styles?.toggleMarginRight || "0" };    
+      left: ${ styles?.toggleLeft || "50%" };    
+      top: ${ styles?.toggleTop || "5px" };    
+      border-bottom: ${ styles?.toggleBorderBottom || "2px solid currentColor" }; 
+      border-right: ${ styles?.toggleBorderRight || "2px solid currentColor" }; 
+      transform: rotate(45deg);
+       
+    }
+
+    .accordion__button[aria-expanded='true']:first-child:after,
+    .accordion__button[aria-selected='true']:first-child:after {
+      transform: rotate(-135deg);
+      top: 10px;    
+    }
+
+    .accordion__panel {    
+      border: ${ styles?.contentBorder || "0" };       
+      border-radius: ${ styles?.contentBorderRadius || "0" }; 
+      margin:  ${ styles?.margin || "0" };
+    }
+  `}  
+`;
+
+const StyledFilterHeader = styled.div`  
```

Comment:
Are all these overrides necessary? I see that a lot of the components have option overrides but are they used? If not, I don't think they should necessarily exist.

Reply:
https://dmitripavlutin.com/use-react-memo-wisely/